Forta-Bots-Challenge-3/package.json

```
@@ -0,0 +1,49 @@
+{
+  "name": "makerdao-bridge-invariant-bot",
+  "displayName": "MakerDAO Bridge Invariant Bot",
+  "version": "0.0.1",
+  "description": "Detects when the MakerDAO Bridge invariant is violated on Arbitrum and Optimism",
+  "longDescription": "This bot checks the DAI balance of L1 escrow contracts and the total supply of L2 DAI contracts in a block to emit an alert when the bridge invariant is violated on Optimism and Arbitrum",
```

```suggestion
  "description": "Detects when the MakerDAO Bridge invariant is violated on Arbitrum and/or Optimism",
  "longDescription": "This bot checks the DAI balance of L1 escrow contracts and the total supply of L2 DAI contracts in a block to emit an alert when the bridge invariant is violated on Optimism and/or Arbitrum",
```
Small fix, as the alerts would be for either or both being violated. Not only when they both are.